src/backend/common/middleware.py

```diff
@@ -66,6 +69,50 @@ def _run_after(self):
             execute_callbacks()
 
 
+def install_backend_security(app: Flask) -> None:
```

Comment:
- [ ] Is there a way to make sure we don't handle `enqueue` tasks if this route hasn't been installed? Seems pretty important. I'm sure we can build something in to `_handle_defer`, right?